webapp/src/components/dot_menu.tsx

```diff
@@ -129,7 +131,7 @@ const DotMenu = ({
                 data-testid='dropdownmenu'
                 onClick={(e) => {
                     e.stopPropagation();
-                    setOpen(false);
+                    return isMultiCheckbox ? null : setOpen(false);
```

Comment:
This return ternary feels a little odd to me. Could we just do an if block?

```suggestion
                   if (closeOnClick) {
                       setOpen(false);
                   }
```

Reply:
resolved [f5bf468](https://github.com/mattermost/mattermost-plugin-playbooks/pull/1467/commits/f5bf468b7f99baa539993767d8dc49f7436ab0a5#diff-cd9f22e13bba5ded76a48757c8918b7ca3e7dd1dc01cd055ba7543d9a64b522bR134-R136)